pkg/minikube/cluster/commands.go

```diff
@@ -1,6 +1,9 @@
 package cluster
 
+import "fmt"
+
 var startCommand = `
 # Run with nohup so it stays up. Redirect logs to useful places.
 PATH=/usr/local/sbin:$PATH nohup sudo /usr/local/bin/localkube start --generate-certs=false > /var/log/localkube.out 2> /var/log/localkube.err < /dev/null &
 `
+var logsCommand = fmt.Sprintf("tail -n +1 %s %s", remoteLocalKubeErrPath, remoteLocalKubeOutPath)
```

Comment:
You could put the constants right in here to make it clearer, once you delete the old code.